src/main/java/bookstore/repositories/order/IOrderRepository.java

```diff
@@ -0,0 +1,17 @@
+package bookstore.repositories.order;
+
+import bookstore.models.Order;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface IOrderRepository extends JpaRepository<Order, Long> {
+    @Query("FROM Order order "
+            + "LEFT JOIN FETCH order.orderItems orderItem "
+            + "LEFT JOIN FETCH order.user "
+            + "LEFT JOIN FETCH orderItem.book "
```

Comment:
```suggestion
            + "JOIN FETCH order.orderItems orderItem "
            + "JOIN FETCH order.user "
            + "JOIN FETCH orderItem.book "
```
Order can't exist without items etc.